content/How to use Lottie Animations on SwiftUI.md

```diff
@@ -0,0 +1,64 @@
+![Main image](images/lottie-cover.png)
+
+# How to use Lottie Animations in SwiftUI
+
+## Introduction
+
+If you're here already, there's a 99.999% chance you know what Lottie is. For those still confused on what this article is about: Lottie is an awesome iOS, Android, and React Native library created by Airbnb that renders After Effects animations in real time, allowing you to use animations as easily as you use static images. You can find more info on their [website](https://airbnb.design/lottie/) and all the technical documentation you want on their [github](https://github.com/airbnb/lottie) page.
+
+
+## Overview
+
+In this tutorial, we’ll create a simple SwiftUI view using `UIViewRepresentable` in order to create and load any Lottie animation into your project.
+
+## Lets get started
+
+### Installation
+
+In order to use Lottie inside your project we need to install the library. For this we have a few different options like Cocapods, Carthage or the one we will use in this tutorial Swift Package Manager.
+
+First of all, on Xcode click on `File` and then select the `Add Package Dependency...` option in Swift Packages. In the input box, enter: `https://github.com/airbnb/lottie-ios.git` (this is the library repository from where Xcode will download the code) and then just hit next and leave the major version thats selected by default to use the more recent release for Lottie.
+
+
+### Hosting Lottie on SwiftUI 
+
+Now that you have Lottie already installed on your project we can start creating the view. For this you'll need to create a new SwiftUI file. Call it `LottieView` and delete everything except the import statement and add the following:
```

Comment:
maybe just create a Swift file and import SwiftUI?

Reply:
When you create a new SwiftUI view the Text("Hello world") and preview are already there. You need to delete that. I can remove that part but I feel like is the exact step you need to do in order to continue with the tutorial 🤔

Reply:
it feels like an unnecessary extra step since you're removing everything beside de import anyways